src/store/metadata/stub/content/datasets.json

```diff
@@ -46,8 +45,51 @@
                 "modified": "2017-02-21T09:30:00Z"
               }
             ]
-        }
+        },
+        {
+          "@id": "https://staging.idpd.uk/datasets/beis-local-authority-greenhouse-gas-emissions",
+          "@type": "dcat:DatasetSeries",
+          "identifier": "local-authority-greenhouse-gas-emissions",
+          "title": "Local authority greenhouse gas emissions",
+          "summary": "UK local authority estimates of greenhouse gas emissions",
+          "description": "These statistics provide a breakdown of greenhouse gas emissions across the UK, using...",
+          "issued": "2023-06-29T00:00:00",
+          "modified": "2023-06-29T00:00:00",
+          "next_release": "2023-07-06T09:30:00Z",
+          "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+          "creator": "https://www.gov.uk/government/organisations/department-for-energy-security-and-net-zero",
+          "theme": "http://gss-data.org.uk/def/gdp#climate-change",
+          "contact_point": {
+            "name": "UK local authority and regional estimates of greenhouse gas emissions",
+            "email": "GreenhouseGas.Statistics@energysecurity.gov.uk"
```

Comment:
this needs to have `mailto:` at the beginning. It's something that regretably our schemas aren't policing at the moment.